.circleci/config.yml

```diff
@@ -0,0 +1,92 @@
+---
+common-steps:
+  - &installdeps
+    run:
+      name: Install Debian packaging dependencies
+      command: make install-deps
+
+  - &fetchwheels
+    run:
+      name: Download wheels and sources
+      command: make fetch-wheels
+
+  - &makesourcetarball
+    run:
+      name: Get latest tag for the project and make a source tarball
+      command: |
+        cd ~/packaging/securedrop-*
+        export LATEST_TAG="$(git describe --tags $(git rev-list --tags --max-count=1))"
```

Comment:
Nice solution on the tag lookup, I'd have gone with `git tag | sort -nr | head -n1`, but this looks more durable. :+1: